README.md

```diff
@@ -24,7 +24,25 @@ See the script for parameters.
 
 # Common Errors
 
-## `FATA[0000] unable to fetch ssh: handshake failed: ssh: unable to authenticate, attempted methods [none publickey], no supported methods remain`
+- Branch errors
+You might see this if your git head's branch is different from the default branch. 
+```
+➜  aws-oidc git:(main) bff bump
+Please only release versions from master.
+SHAs on branches could go away if a branch is rebased or squashed.
+latestVersionTag "0.22.3+96d6e06"
+fileversion "0.22.3"
+FATA[0000] tag does not match VERSION file              
+make: *** [release] Error 1
+```
+To change the default branch in `head` from master to main, run this:
+```
+git remote set-head origin main
+```
+Then try running the bff command again. Make sure the default branch is clean!
```

Comment:
@mbarrien I thought this was an okay band-aid approach to the problem you commented about before. What do you think?

Reply:
Ah, didn't see the readme when I was looking at that.

Reply:
Update this to note that you can update the head branch by running `git remote set-head origin -a` instead of asking the user to manually set the branch.

Even better, maybe we should run the equivalent command by default every time we run bff (possibly enable/disable via flag?) Would want to get feedback from others before you do this one.

Reply:
Wait, what's the risk of running `git remote set-head origin -a` every time?

Reply:
It's so convenient!